### DISCUSSION TITLE
[Feature-request] Plans for other CI platforms

### DISCUSSION THREAD

**Discussion by dimitar-hristov** [github/gh-actions-importer#89] [Feature requests] [answered]
Hello :wave: I was wondering if there are any plans for expanding the project to include more CI platforms. I am particularly interested if there are plans for the the importer to support Drone files?

**Comment by ethanis** [comment, marked as answer]
@dimitar-hristov we are working on another migration path that we aren't quite ready to communicate yet (it's not Drone CI, however). We'll be publishing new features and migration paths to the public GitHub [roadmap](https://github.com/orgs/github/projects/4247/views/1) moving forward 👍

**Comment by RinatS** [comment]
Are these roadmap items going to be labeled under "Actions" or else? I do not see anything pertaining to GAI specifically at this point.
